infra/azure-peerpods/vars.tf

```diff
@@ -2,14 +2,26 @@ variable "name_prefix" {
   type = string
 }
 
-variable "image_resource_group_name" {
+variable "resource_group" {
   type = string
 }
```

Comment:
Maybee we can just derive this from `name_prefix`?

Reply:
I wanted the prefix to be randomizable while using a shared group (e.g. `contrast-ci`). Do you think it makes sense to do this within Terraform? It's currently in `test-peerpods.sh`.

Reply:
no, we cannot do this in terraform as it should not happen for developers. Maybe we can do this via justfile.env (azure_resource_group)?

Reply:
Maybe we want to even get rid of `azure_resource_group` and `namespace_suffix` and instead have a single source for naming, like `base_name`.